From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Samuel Attard <samuel@Samuels-Mac.local>
Date: Mon, 6 Jul 2020 14:47:55 -0700
Subject: remove some deps that do not work on arm64

Once these deps have newly built arm64 versions we can remove this patch.

diff --git a/gsutil.vpython b/gsutil.vpython
index b0a646271eae7426988bbef47a4162f47872c1d1..d9465adeb29f1c696d8ef442a753e6737627bd6f 100644
--- a/gsutil.vpython
+++ b/gsutil.vpython
@@ -25,19 +25,19 @@ wheel <
 # "gsutil" on non-GCE can require PyOpenSSL, which, in turn, requires
 # "cryptography".
 
-wheel: <
-  name: "infra/python/wheels/pyopenssl-py2_py3"
-  version: "version:17.2.0"
->
+#wheel: <
+#  name: "infra/python/wheels/pyopenssl-py2_py3"
+#  version: "version:17.2.0"
+#>
 
 ##
 # BEGIN "cryptography" dependencies.
 ##
 
-wheel: <
-  name: "infra/python/wheels/cryptography/${platform}_${py_python}_${py_abi}"
-  version: "version:2.0.3"
->
+#wheel: <
+#  name: "infra/python/wheels/cryptography/${platform}_${py_python}_${py_abi}"
+#  version: "version:2.0.3"
+#>
 
 wheel: <
   name: "infra/python/wheels/appdirs-py2_py3"
@@ -54,10 +54,10 @@ wheel: <
   version: "version:1.1.6"
 >
 
-wheel: <
-  name: "infra/python/wheels/cffi/${platform}_${py_python}_${py_abi}"
-  version: "version:1.10.0"
->
+#wheel: <
+#  name: "infra/python/wheels/cffi/${platform}_${py_python}_${py_abi}"
+#  version: "version:1.10.0"
+#>
 
 wheel: <
   name: "infra/python/wheels/idna-py2_py3"
